.gitignore

```diff
@@ -38,3 +38,7 @@ tmp/
 .cache
 .flask_cache/
 .pytest_cache/
+
+# JS beasts
+public/canvas/canvas-customization.js
+public/canvas/index.js
```

Comment:
@mlincoln - You do NOT want to use `.gitignore` for the purpose of do-not-lint. I would remove the lines above. Instead, modify the `.eslintrc.js` file with guidance from https://eslint.org/docs/latest/use/configure/ignore

Reply:
Got it. If my change to .eslintrc.js satisfies travis, I'll squash the double commit.